src/TransactionReceipt.ts

```diff
@@ -5,26 +5,50 @@ import { FileId } from "./file/FileId";
 import { ExchangeRateSet, exchangeRateSetToSdk } from "./ExchangeRate";
 import { Status } from "./Status";
 
-export interface TransactionReceipt {
-    status: Status;
-    accountId: AccountId | null;
-    fileId: FileId | null;
-    contractId: ContractId | null;
-    exchangeRateSet: ExchangeRateSet | null;
-}
+export class TransactionReceipt {
+    public readonly status: Status;
+    private accountId: AccountId | null;
```

Comment:
Use underscore prefixes for private fields

Reply:
Resolved in #147